### ISSUE TITLE
Compatible with linux?

### ISSUE THREAD

**Issue by fitocuan** [pawankumargurav/kukarsiserver#2]
Hi, I found your code really helpful for my application. How ever I am planning to use linux as my OS. Would this be a problem?

**Comment by pawankumardev** [comment]
Yes fitocuan, you can run same code on any machine having Python Interpreter.
